pages/linux/lid.md

```diff
@@ -0,0 +1,9 @@
+# lid
+
+> Query ID database and report results.
+> On Fedora and Arch Linux, the binary name `lid` is taken by another program. See `tldr libuser-lid`.
+> More information: <https://www.gnu.org/software/idutils/>.
+
+- This page is currently a redirection stub (if you are familiar with this version of this command, please open a pull request!):
```

Comment:
```suggestion
- View documentation for `libuser-lid` (this page is currently a redirection stub; if you are familiar with this version of this command, please open a pull request!):
```
The tldr-bot is throwing an error because "this" ends with the letter "s"

Reply:
I opted not to use the wording "this version of this command", because it can easily be misunderstood. These are two completely different programs that just so happen to have a naming conflict.